packages/components/Button/src/PrimaryButton/PrimaryButton.settings.win32.ts

```diff
@@ -0,0 +1,50 @@
+import { IButtonType } from '../Button.types';
+import { IComposeSettings } from '@uifabricshared/foundation-compose';
+
+export const settings: IComposeSettings<IButtonType> = [
+  {
+    tokens: {
+      backgroundColor: 'brandBackground',
+      color: 'neutralForegroundInverted',
+      borderColor: 'brandBackground',
+    },
+    // TODO: neutralForegroundInverted is not working for icon color.
+    endIcon: {
+      color: '#fff',
+    },
+    icon: {
+      color: '#fff',
+    },
```

Comment:
For consistency sake, can we use a 6 letter hex value?

Reply:
fixed